src/index.html

```diff
@@ -20,6 +20,7 @@ <h1 id="home">base.camp Luftdaten</h1>
         <button id="openbtn" class="openbtn">&#9776;</button>
     </div>
     <div id="about" class="textbox-layer">
+        <button id="tb-close" class="tb-close">&#10005</button>
```

Comment:
IDs should be unique across the HTML. There are two `id="tb-close"` now. I think you could just remove them entirely and just use the classes (see my comment further down).